src/org/usfirst/frc/team1492/robot/autonomous/MissionScript.java

```diff
@@ -0,0 +1,102 @@
+package org.usfirst.frc.team1492.robot.autonomous;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+
+public class MissionScript {
+    
+    public static Mission parseMission(int id, List<String> code, CommandFactory factory) {
+        Mission mission = new Mission(id);
+        int lineIndex = 0;
+        for (String line : code) {
+            String lineWithoutComment = line.split("//")[0];
+
+            String[] parenSplit = lineWithoutComment.split("\\(");
+            if (parenSplit.length == 2) {
+                String name = parenSplit[0].trim();
+                String signature = name + "(";
+                String[] parametersTogether = parenSplit[1].split("\\)");
+                
+                Class<?>[] parameterClasses;
+                Object[] parameterValues;
+                
+                if(parametersTogether.length > 0){
+                    String[] parameters = parametersTogether[0].split(",");
+                    int paramIndex = 0;
+                    parameterClasses = new Class[parameters.length];
+                    parameterValues = new Object[parameters.length];
+                    for (String parameter : parameters) {
+                        if (parameter.equals("true")) {
+                            parameterClasses[paramIndex] = boolean.class;
+                            parameterValues[paramIndex] = true;
+                        } else if (parameter.equals("false")) {
+                            parameterClasses[paramIndex] = boolean.class;
+                            parameterValues[paramIndex] = false;
+                        } else {
+                            try {
+                                parameterClasses[paramIndex] = double.class;
+                                parameterValues[paramIndex] = Double.parseDouble(parameter);
+                            } catch (NumberFormatException e) {
+                                System.err.println("Failed to parse parameter as boolean or double: \""
+                                        + parameter + "\"");
+                                return null;
```

Comment:
We should throw an exception here.
A `ParseException` or rethrowing the `NumberFormatException` is probably appropriate.